server/athenian/api/controllers/features/everything.py

```diff
@@ -23,14 +23,15 @@
 from athenian.api.controllers.miners.github.deployment_light import fetch_repository_environments
 from athenian.api.controllers.miners.github.developer import DeveloperTopic, \
     mine_developer_activities
-from athenian.api.controllers.miners.github.precomputed_prs import \
-    DonePRFactsLoader, MergedPRFactsLoader, OpenPRFactsLoader
+from athenian.api.controllers.miners.github.precomputed_prs import DonePRFactsLoader, \
+    MergedPRFactsLoader, OpenPRFactsLoader
```

Comment:
I see that you've reformatted the imports. Can you please extract this formatting in a separate PR :pray: